stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/DeletedMessageListItemPredicate.kt

```diff
@@ -22,6 +22,15 @@ import io.getstream.chat.android.client.ChatClient
 /**
  * Predicate class used to filter [MessageListItem.MessageItem] items which are deleted. Used by [MessageListView.setDeletedMessageListItemPredicate].
  */
+@Deprecated(
+    message = "Deprecated in favor of filtering in the ViewModel by using " +
+        "'io.getstream.chat.android.common.state.DeletedMessageVisibility'" +
+        " together with 'com.getstream.sdk.chat.viewmodel.messages.setDeletedMessageVisibility()'",
```

Comment:
`MessageListViewModel.setDeletedMessageVisibility`?

Reply:
I'm genuinely not sure what the convention here is so I went via the route of being explicit in order to make people able to locate the correct import.

Reply:
Well spotted